src/App.tsx

```diff
@@ -1,61 +1,122 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+const newTodoId = Math.max(...todosFromServer.map(todo => todo.id)) + 1;
```

Comment:
This function launches just once 
<img width="451" alt="Screenshot 2024-02-03 at 13 44 03" src="https://github.com/mate-academy/react_add-todo-form/assets/135232001/8c73f2c0-f8e3-4bdd-a2c7-a3b8af2b235f">
<img width="593" alt="Screenshot 2024-02-03 at 13 53 20" src="https://github.com/mate-academy/react_add-todo-form/assets/135232001/fba17f2a-b0f5-4ea3-8a23-189dff53b543">